sdk/python/express_relay/models/svm.py

```diff
@@ -399,6 +402,14 @@ def check_result(self):
             ), "bid result should not be empty when status is not pending or lost"
         return self
 
+    @model_validator(mode="after")
+    def check_failed_reason(self):
+        if self.type == BidStatusVariantsSvm.SUBMISSION_FAILED:
+            assert (
```

Comment:
q: for backwards compat reasons, we might want to allow reason to be null right? not sure if its populated for older data